src/main/java/uk/gov/justice/digital/hmpps/whereabouts/services/Elite2ApiHealth.java

```diff
@@ -22,7 +22,7 @@ public Elite2ApiHealth(@Qualifier("elite2ApiHealthRestTemplate") final RestTempl
     @Override
     public Health health() {
         try {
-            final var responseEntity = this.restTemplate.getForEntity("/health", String.class);
+            final var responseEntity = this.restTemplate.getForEntity("/ping", String.class);
```

Comment:
move to `/ping` as would get caught in endless loop with `/health` as elite on same port as whereabouts.  Needs some more thought as it would be good for all apps to be able to run together out of the box